webapp/src/lib/components/spot-component/spot-display.svelte

```diff
@@ -4,35 +4,33 @@
     export let spot;
 </script>
 
-<a href={`your-spots/${spot.id}/spot-info`}>
-    <div class="spot-info">
+<div class="spot-info">
+    <div>
+        <img src={BackgroundImage} class="spot-image" alt="A parking spot" />
+    </div>
+    <div>
         <div>
-            <img src={BackgroundImage} class="spot-image" alt="A parking spot" />
+            <p class="spot-title">
+                {spot.location.street_address}, {spot.location.postal_code}
+                {#if spot.isListed}
+                    <Tag type="green" style="font-size: 1rem;">listed</Tag>
+                {/if}
+            </p>
+            <span style="font-size: 1rem"
```

Comment:
Please put the style in the style block.

Reply:
This is a Carbon design quirk. Only inline-style works